package.json

```diff
@@ -20,13 +20,16 @@
   "bugs": {
     "url": "https://github.com/meandavejustice/min-vid/issues"
   },
+  "config": {
```

Comment:
Not a change you made in this PR, but the version `0.3.3b` should be `0.3.3-b` to follow semver.